apps/src/templates/export/expo/App.js.ejs

```diff
@@ -56,6 +56,16 @@ export default class App extends React.Component {
     });
   };
 
+  onLoadEnd = () => {
+    requestAnimationFrame(() => {
+      requestAnimationFrame(() => {
+        this.setState({
+          loaded: true,
```

Comment:
This looks like a hack.  Worth a comment explaining why two animation frames is sufficient to consider the app loaded?